src/scanner.coffee.md

```diff
@@ -139,14 +139,15 @@ of cells or null if the scanner is exhausted.
         return callback() if response and response.statusCode is 204
         return callback err if err
         cells = []
+        encoding = if @options.encoding is 'undefined' then @options.encoding else @client.options.encoding
```

Comment:
then let remove the scanner.option-part and have a fallback for client.option and that's it.

Btw: thanks for the whole package, it helps us a lot!